cordova-lib/spec-cordova/lazy_load.spec.js

```diff
@@ -26,7 +26,8 @@ var lazy_load = require('../src/cordova/lazy_load'),
     request = require('request'),
     fs = require('fs'),
     Q = require('q'),
-    platforms = require('../src/cordova/platforms');
+    platforms = require('../src/cordova/platforms'),
```

Comment:
The first several screens of this diff are no-op changes. Please have a look through the diff yourself and undo any highlighted lines that maybe your editor is changing on you? Might be easier to start fresh and re-apply your change rather than try and fix up this one.